app/views/tools/show.html.erb

```diff
@@ -1,6 +1,58 @@
-<h1> Display tools with a specifc id </h1>
+<link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
+<script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
+<!------ Include the above in your HEAD tag ---------->
+
+<!DOCTYPE html>
```

Comment:
It's not necessary to use the HTML, body and head tags cause they are already in the main layout `application.html.erb`